src/Projekt/Projekt.fsproj

```diff
@@ -81,16 +57,54 @@
   <Target Name="BeforeBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Templates Include="$(SolutionDir)\templates\**\*.*"/>
-  </ItemGroup>
   <Target Name="AfterBuild">
-    <Copy
-        SourceFiles="@(Templates)"
-        DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)"
-        />
+    <Copy SourceFiles="@(Templates)" DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)" />
```

Comment:
Crap, this happened because the templates node caused me to be unable to build in VS2015.  Do you all have this same problem?

![image](https://cloud.githubusercontent.com/assets/573979/9042735/71a4ecd8-39d6-11e5-8b84-f869380bf59a.png)

(it may be a 2015 problem)